packages/core/src/mechanic-drawloop.js

```diff
@@ -0,0 +1,113 @@
+let instance,
+  isPlaying = false,
+  frameRate,
+  fpsInterval,
+  frameCount,
+  lastFrameTime,
+  raf,
+  callback;
+
+const epsilon = 5;
+
+class MechanicDrawloop {
+  /**
+   * Drawloop constructor. The drawloop is realized as a singleton. Making sure
+   * there is only one instance of the drawloop is needed so we can easily clear
+   * any pending animation frames when it's time to update a design function due
+   * to code changes or new input values.
```

Comment:
Yes, Mechanic Core adds a `this.drawLoop.stop()`. I mean technically you shouldn't be able to get a second instance of drawloop, since we export and instance and not the class.